src/plugins/Makefile.am

```diff
@@ -128,9 +129,18 @@ libgs_plugin_dpkg_la_CFLAGS = $(GS_PLUGIN_CFLAGS) $(WARN_CFLAGS)
 if HAVE_EXTERNAL_APPSTREAM
 libgs_plugin_external_appstream_la_SOURCES =		\
 	gs-plugin-external-appstream.c
-libgs_plugin_external_appstream_la_LIBADD = $(GS_PLUGIN_LIBS)
+libgs_plugin_external_appstream_la_LIBADD = $(GS_PLUGIN_LIBS) $(SOUP_LIBS)
 libgs_plugin_external_appstream_la_LDFLAGS = -module -avoid-version
 libgs_plugin_external_appstream_la_CFLAGS = $(GS_PLUGIN_CFLAGS) $(WARN_CFLAGS)
+
+gnome-software-install-appstream: gnome-software-install-appstream.in Makefile
+	sed -e "s|@localstatedir[@]|$(localstatedir)|g" $< > $@
+
+CLEANFILES =				\
+	gnome-software-install-appstream
+
+dist_libexec_SCRIPTS =			\
+	gnome-software-install-appstream
```

Comment:
2 notes:

1. This shouldn't be distributed as you want it to be generated during the build with the configured setting for `localstatedir`. So, you should drop the `dist_` prefix.

2. As noted on the previous commit, this should be part of the conditional around `plugin_LTLIBRARIES` and the rest of these rules (the `sed` and the `CLEANFILES` setting) should be unconditional.